### ISSUE TITLE
Lint/UnusedMethodArgument offense ignored with comment body

### ISSUE THREAD

**Issue by garettarrowood** [bbatsov/rubocop#4850]
## Expected behavior

I expect to see a `Lint/UnusedMethodArgument` offense in the following code snippet:

```ruby
# frozen_string_literal: true

# A Fooy example
class Foo
  def unused_arg(arg)
    # Fooy comment
  end
end
```

Interestingly, when the method is called on `self`, the offense is correctly raised.

```ruby
# frozen_string_literal: true

# A Fooy example
class Foo
  def self.unused_arg(arg)
    # Fooy comment
  end
end
```

This may be related to #4677 , simply because it deals with the same cop.

## Actual behavior

No offense is raised by first example.

## Steps to reproduce the problem

Run rubocop on this snippet:

```ruby
# frozen_string_literal: true

# A Fooy example
class Foo
  def unused_arg(arg)
    # Fooy comment
  end
end
```

## RuboCop version

```shell
0.50.0 (using Parser 2.4.0.0, running on ruby 2.3.3 x86_64-darwin15)
```

**Comment by Drenmi** [comment]
There's a configuration option, `IgnoreEmptyMethods`, which defaults to `true`. If you set this to `false`, it will report an offense on the method definition.

The disparity between an instance method- and class method definition looks like a bug though.
